web/themes/custom/atrium/src/js/jsBackButton.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file
+ * Add back button functionality for .js-back-button.
+ */
+
+(function($, Drupal) {
+  "use strict";
+
+  Drupal.behaviors.jsBackButton = {
+    attach: function(context) {
+      // Define button.
+      const button = $(".js-back-button");
+
```

Comment:
Can we add this:
```
$(button).attr('href', '#');
```
When in the wysiwyg, I put in an actual URL. Then on the front end, I was confused since the browser status bar showed the URL even though the click does go back. I don't know if the editors will remember to put in a '#' in the URL space in ckeditor.